src/cmd-buildextend-live

```diff
@@ -415,6 +415,10 @@ def generate_iso():
         # can be EFI/fedora or EFI/redhat
         grubpath = glob.glob(os.path.join(tmpisoroot, 'EFI/*/grub.cfg'))
         shutil.move(grubpath[0], os.path.join(tmpisoroot, 'boot/grub/grub.cfg'))
+        for key in list(files_with_karg_embed_areas):
+            if re.search(r"/EFI/.*/grub.cfg", key):
```

Comment:
Can't we just do a direct comparison here instead? E.g.

```suggestion
            if key == grubpath:
```

?

Reply:
Thanks for all suggestion folks.  Adding those updates and performing more testing

Reply:
This doesn't look resolved.

Reply:
@mrcxavier this is still outstanding; could you please test the suggestion and report back?

Reply:
Applied and testing was successful